oscar/run_vqa.py

```diff
@@ -36,7 +36,7 @@
 
 log_json = []
 debug_size = 500
-
+local_rank = int(os.getenv('LOCAL_RANK','0'))
```

Comment:
you remove args.local_rank and switch to an env var. That breaks people that have been using args.local_rank and will create trouble if we ever try to merge this repo upstream.
If you'd change to:
```
int(os.getenv('LOCAL_RANK', str(args.local_rank)))
```
and add the args.local_rank back you can support both.